edx_notifications/server/web/static/edx_notifications/js/vendor/dev/backbone.js

```diff
@@ -0,0 +1,1608 @@
+//     Backbone.js 1.1.2
```

Comment:
It seems a shame to have another copy of Backbone (and other third-party libraries) here. Could you use RequireJS to declare it as a dependency and just assume that it will always be deployed somewhere that already has Backbone installed?